### ISSUE TITLE
Add support to ktlint

### ISSUE THREAD

**Issue by iamsh4shank** [openMF/android-client#1830]
**Summary:** 

As the project is also ported to Kotlin and the new code is written in Kotlin, so adding Ktlint would be good option.

**Comment by duttabhishek0** [comment]
Can I work on this?

**Comment by danishjamal104** [comment]
> Can I work on this?

Currently we are more focus on releasing newer version of the app, and lot of code conversion is left which may cause errors so currently we are moving ahead with the legacy check style and pmd rules. Better you work on some issues that are related to the new release, Thanks :)

**Comment by duttabhishek0** [comment]
Sure I would love to. So one question, the issues which are related to the new release would be available here only, right?

**Comment by iamsh4shank** [comment]
@duttabhishek0 yes they are available here, meanwhile, you can also build the project and test the app if there are some bugs or issues.

**Comment by parjanya-rajput** [comment]
Is this issue still open, if yes can I work on it?

**Comment by therajanmaurya** [comment]
We have Deket support and Klint already so closing it
